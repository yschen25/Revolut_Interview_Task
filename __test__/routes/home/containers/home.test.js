import React from 'react';
import { render } from '@testing-library/react';
import Home from '../../../../src/routes/home/containers/home';

describe('test <Index />', () => {
    const { container, getByTestId } = render(<Home />);

    it('<Index /> snapshot', () => {
        expect(container.firstChild).toMatchSnapshot();
    });

    it('test <Index /> and find title', () => {
        const elem = getByTestId('display_title');

        expect(elem.textContent).toBe('Welcome to React-Redux-App!');
    });
});
