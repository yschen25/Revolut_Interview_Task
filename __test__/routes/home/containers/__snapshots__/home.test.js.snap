// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test <Index /> <Index /> snapshot 1`] = `
<div
  class="sc-bdVaJa jggxrB"
>
  <div
    class="sc-bwzfXH jTZVNk"
  />
  <div
    class="sc-bxivhb kgXhAB"
  />
  <div
    class="sc-bwzfXH hHhAFc"
  />
  <h1
    class="sc-htpNat jUnqYe"
    data-testid="display_title"
  >
    Welcome to React-Redux-App!
  </h1>
</div>
`;
